resources/views/components/banner.blade.php

```diff
@@ -9,7 +9,8 @@
                     message = event.detail.message;
                     show = true;
                 });
-            ">
+            "
+            style="display: none;">
```

Comment:
Please use a tailwind class here.

Reply:
It is not possible to use `hidden` class without another changes as it will be hidden forever.

To make `class="hidden"` work, I will need to update `:class` attribute and add `, hidden: !show || !message`. Otherwise apline.js does not know, that he should remove `hidden` class.

If it is ok, I will update this PR, just let me know.

Reply:
@RiKap in app.cas add the following:

```css
[x-cloak] {
    display: none;
}
```

You can then use this as an HTML attribute! 😎

Reply:
@RiKap ah okay, I was confusing this with Vue 3's behavior for a sec 😅 Just leave this be for now.